lib/lightning/helpers.ex

```diff
@@ -60,6 +60,10 @@ defmodule Lightning.Helpers do
     data |> Map.from_struct() |> json_safe()
   end
 
+  def json_safe(%{microsecond: _} = map) do
```

Comment:
@stuartc and this too. I am plucking out the microsecond tuple to allow json_safe not to break